Source/Fuse.Marshal/Marshal.Cast.uno

```diff
@@ -114,6 +114,75 @@ namespace Fuse
 
 			throw new MarshalException(o, typeof(float4));
 		}
+		
+		/**
+			Converts a value to a float4
+			
+			@return true if converted successfully, false if no suitable conversion exists
+			@param value the result value (0 padded as necessary)
+			@param size the size of the result
+		*/
+		public static bool TryMarshalToFloat4(object o, out float4 value, out int size)
```

Comment:
Right, makes sense from an implementation point of view.

But, do we really want different behavior for different usages here? This sounds pretty confusing to the end-user to me...